src/sentry/static/sentry/app/components/events/interfaces/keyValueList.jsx

```diff
@@ -31,7 +31,7 @@ class KeyValueList extends React.Component {
       data = Object.keys(data).map(key => [key, data[key]]);
     }
 
-    data = this.props.isSorted ? _.sortBy(data, [(key, value) => key]) : data;
+    data = this.props.isSorted ? _.sortBy(data, [([key]) => key]) : data;
```

Comment:
data is a list of tuples, so previously it was sorting by both values of the "tuple" and would break if the 2nd value of tuple was not a string.